server/services/AuthService.js

```diff
@@ -12,6 +16,12 @@ class AuthService {
     if (hash === walletObject.password) {
       const token = JWTService.sign(walletObject);
 
+      await event.logEvent({
+        loggedInWalletId: walletObject.id,
+        type: 'login',
```

Comment:
Let's make type an enum in the code itself

Reply:
Created an EventEnums